test/test_xhistogram.cpp

```diff
@@ -38,6 +38,16 @@ namespace xt
             EXPECT_EQ(count(0), 2.);
             EXPECT_EQ(count(1), 2.);
         }
+
+        xt::xarray<double> arr = {1., 1., 2., 2.};
```

Comment:
I would put this in the same scope as the rest (between `{...}` just below.

Reply:
Fixed! Thanks for the review.